### DISCUSSION TITLE
tutorials/3d/using_decals

### DISCUSSION THREAD

**Discussion by giscus** [godotengine/godot-docs-user-notes#94] [User-contributed notes]
# tutorials/3d/using_decals

Decals are projected textures that apply on opaque or transparent surfaces in 3D. This projection happens in real-time and doesn't rely on mesh generation. This allows you to move decals every fram...

https://docs.godotengine.org/en/stable/tutorials/3d/using_decals.html

**Comment by DiskoTecha** [comment]
The 3D decals demo link is broken. Here is the correct link: [3D Demo Projects](https://github.com/godotengine/godot-demo-projects/tree/master/3d/decals)

**Reply by Calinou** [reply]
- Fixed in https://github.com/godotengine/godot-docs/pull/9864.

**Comment by PurpBatBoi** [comment]
Someone really should on making decals compatible with Compatibility.

**Reply by Calinou** [reply]
Please open a [proposal](https://github.com/godotengine/godot-proposals) for this, as feature proposals should be tracked on the Godot proposals repository.

**Comment by 23elemental23** [comment]
The decals feel fairly limited. Unity lets you use a custom material shader in your decals, godot wont let you repeat a texture in a tiled pattern without placing a million decals one by one. I like godot, I hope decals get better and can eventually get used in the compatibility renderer.

**Reply by Calinou** [reply]
See these proposals:

- https://github.com/godotengine/godot-proposals/issues/4938
- https://github.com/godotengine/godot-proposals/discussions/12903 (an add-on is linked there too)
